test/e2e/restorebackup_test.go

```diff
@@ -130,10 +95,48 @@ func createJenkinsWithBackupAndRestoreConfigured(t *testing.T, name, namespace s
 					},
 				},
 			},
+			GroovyScripts: v1alpha2.GroovyScripts{
+				Customization: v1alpha2.Customization{
+					Configurations: []v1alpha2.ConfigMapRef{},
+				},
+			},
+			ConfigurationAsCode: v1alpha2.ConfigurationAsCode{
+				Customization: v1alpha2.Customization{
+					Configurations: []v1alpha2.ConfigMapRef{},
+				},
+			},
 			Master: v1alpha2.JenkinsMaster{
 				Containers: []v1alpha2.Container{
 					{
 						Name: resources.JenkinsMasterContainerName,
+						ReadinessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   "/login",
+									Port:   intstr.FromString("http"),
+									Scheme: corev1.URISchemeHTTP,
+								},
+							},
+							InitialDelaySeconds: int32(80),
+							TimeoutSeconds:      int32(4),
+							FailureThreshold:    int32(10),
+							SuccessThreshold:    int32(1),
+							PeriodSeconds:       int32(1),
+						},
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   "/login",
+									Port:   intstr.FromString("http"),
+									Scheme: corev1.URISchemeHTTP,
+								},
+							},
+							InitialDelaySeconds: int32(80),
+							TimeoutSeconds:      int32(4),
+							FailureThreshold:    int32(10),
+							SuccessThreshold:    int32(1),
+							PeriodSeconds:       int32(1),
+						},
```

Comment:
please remove, the default provides by the operator should be used

Reply:
If I do there is a loop of setting new probes until everything crashes. I wonder if it is a problem beyond test?

Reply:
Maybe it is a problem with setting defaults. Please look at my commits I've made some changes related to it.